tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -1,4 +1,5 @@
 def improper_indents(indents):
+
```

Comment:
How? I think this issue is still valid... Based on this https://github.com/coala/coala-bears/pull/699#discussion_r98310815
I and Sanket agreed to have a blank line before and after docstring.